site/pages/create-consequence-services.page.tsx

```diff
@@ -0,0 +1,542 @@
+                            stateUpdater={stateUpdater}
+                            value={pageState.inputs.disruptionSeverity}
+                            initialErrors={pageState.errors}
+                            schema={stopsConsequenceSchema.shape.disruptionSeverity}
+                        />
+
+                        <Radios<ServicesConsequence>
+                            display="Direction of disruption"
+                            displaySize="l"
+                            radioDetail={[
+                                {
+                                    value: "allDirections",
+                                    display: "All directions",
+                                },
+                                {
+                                    value: "inbound",
+                                    display: "Inbound",
+                                },
+                                {
+                                    value: "outbound",
+                                    display: "Outbound",
+                                },
+                            ]}
+                            inputName="disruptionDirection"
+                            stateUpdater={stateUpdater}
+                            value={pageState.inputs.disruptionDirection}
+                            initialErrors={pageState.errors}
+                            schema={servicesConsequenceSchema.shape.disruptionDirection}
+                        />
+
+                        <input type="hidden" name="consequenceType" value="services" />
+                        <input
+                            type="hidden"
+                            name="vehicleMode"
+                            value={previousConsequenceInformation.modeOfTransport}
+                        />
+
+                        <button className="govuk-button mt-8" data-module="govuk-button">
+                            Save and continue
+                        </button>
+                    </div>
+                </>
+            </form>
+        </BaseLayout>
+    );
+};
+
+export const getServerSideProps = async (ctx: NextPageContext): Promise<{ props: object } | void> => {
+    let inputs: PageState<Partial<StopsConsequence>> = {
+        errors: [],
+        inputs: {},
+    };
+
+    let previousConsequenceInformationData = {};
+
+    const cookies = parseCookies(ctx);
+    const typeCookie = cookies[COOKIES_CONSEQUENCE_TYPE_INFO];
+    const dataCookie = cookies[COOKIES_CONSEQUENCE_INFO];
+    const errorCookie = cookies[COOKIES_CONSEQUENCE_SERVICES_ERRORS];
+
+    if (typeCookie) {
+        const previousConsequenceInformation = typeOfConsequenceSchema.safeParse(JSON.parse(typeCookie));
+
+        if (previousConsequenceInformation.success) {
+            previousConsequenceInformationData = previousConsequenceInformation.data;
+        }
+    }
+
+    inputs = getPageStateFromCookies<StopsConsequence>(dataCookie, errorCookie, stopsConsequenceSchema);
```

Comment:
Anywhere with `StopsConsequence` should be replaced with `ServicesConsequence` and same with `stopsConsequenceSchema` to `servicesConsequenceSchema`

Reply:
done